src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -224,9 +224,18 @@ private void ExitTour()
                 {
                     tmpStep.StepClosed -= Popup_StepClosed;
                 }
+
                 string guidName = Resources.ResourceManager.GetString(currentGuide.GuideNameResource).Replace("_", "");
-                Logging.Analytics.TrackEvent(Logging.Actions.End, Logging.Categories.GuidedTourOperations, guidName, currentGuide.SequenceOrder);
+                if (currentGuide.TotalSteps - 1 == currentGuide.CurrentStep.Sequence)
+                {
+                    Logging.Analytics.TrackEvent(Logging.Actions.Completed, Logging.Categories.GuidedTourOperations, guidName, currentGuide.CurrentStep.Sequence);
+                }
+                else 
+                {
+                    Logging.Analytics.TrackEvent(Logging.Actions.End, Logging.Categories.GuidedTourOperations, guidName, currentGuide.CurrentStep.Sequence);
```

Comment:
@QilongTang I updated the last argument from `SequenceOrder` to CurrentStep.Sequence`, this will tell us at which step the user ended the tour rather than which tour was ended. I think we can rely on tour name for that. Let me know your thougths

Reply:
I agree